multimodal/src/autogluon/multimodal/models/timm_image.py

```diff
@@ -13,6 +13,10 @@
 logger = logging.getLogger(AUTOMM)
 
 
+# Stores the
+SUPPORT_VARIABLE_INPUT_SIZE_TIMM_CLASSES = {"convnext", "efficientnet", "mobilenetv3", "regnet", "resnet"}
```

Comment:
Just curious, does this list need to manually update when there are more models support variable input size?

Reply:
Yes. I implemented two logics: 1) if the test_input_size is different from input_size, this means that the network allows variable input size; 2) stored some commonly used backbones that supports variable input size.